### PR TITLE
Adding February 2024 update

### PR DESCRIPTION
**Pull request by duelinggalois**
- Added update
- Fixed desmos on overview
- Fixed ordering in docs

### CODE REVIEW

File: docs/updates/February_2024.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 4
+---
+
+# February 2024
+
+## tl;dr
+
+- Hired a solidity developer
+- Completed an Audit of the draft white paper with ChainSecurity
```
Comment (Uhan19) [inline docs/updates/February_2024.md:10]: NIT: lower case audit?

File: docs/updates/February_2024.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+on me and my time being spread thin while raising seed, Imi will take a huge
+load off me and ensure we solve handle the most technical problems efficiently.
+
+### Contracts
+
+This last month we contracted with ChainSecurity to review a draft of our
+Whitepaper. I did this to make sure we were getting an outside technical
+feedback on what we were building. I am excited that there were no major gotchas
+that came out of the review. We did realize there were some assumptions we made
+about block times that do not hold on all L2s that will requires a small fix to
```
Comment (Uhan19) [inline docs/updates/February_2024.md:35]: "require:

**Review by 1m1-github — APPROVED** [review]
looks good

**Review by cirethic — APPROVED** [review]
lg

**Pull request merged**